pkg/network/ebpf/c/netns.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __NETNS_H
+#define __NETNS_H
+
+#if defined(COMPILE_CORE) || defined(COMPILE_RUNTIME)
+
+#include "bpf_core_read.h"
+#include "bpf_telemetry.h"
+
+#ifdef COMPILE_RUNTIME
+#include <net/net_namespace.h>
+#include <net/sock.h>
+#endif
+
+#ifdef COMPILE_CORE
+#define sk_net __sk_common.skc_net
+#define CONFIG_NET_NS
+#endif
+
+static __always_inline u32 get_netns_from_ct_net(struct net* ct_net) {
+    u32 net_ns_inum = 0;
+#ifdef CONFIG_NET_NS
+#if defined(_LINUX_NS_COMMON_H) || defined(COMPILE_CORE)
+    BPF_CORE_READ_INTO(&net_ns_inum, ct_net, ns.inum);
```

Comment:
This will need to be improved for the CO-RE version to work on older kernels. Not required for this PR though.